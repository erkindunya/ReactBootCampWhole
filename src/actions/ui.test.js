import {
  toggleMessageDetail,
  TOGGLE_MESSAGE_DETAIL,
} from './ui'

describe('Thread action', () => {
  it(``, () => {
    
  })
})
