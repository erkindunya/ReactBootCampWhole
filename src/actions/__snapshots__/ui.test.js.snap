// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thread action should return an action of type TOGGLE_MESSAGE_DETAIL 1`] = `
Object {
  "type": "TOGGLE_MESSAGE_DETAIL",
}
`;
